pontoon/base/migrations/0094_entity_document.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-06-02 09:17
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import TextField, Value
+from django.db.models.functions import Concat
+
+from pontoon.db.migrations import (
+    MultiFieldTRGMIndex,
+)
+
+
+def set_entity_document(apps, schema_editor):
+    Entity = apps.get_model('base', 'Entity')
+    Entity.objects.update(
+        document=Concat('key', Value(' '), 'string', Value(' '), 'string_plural', Value(' '), 'comment')
+    )
+
+entity_document_update_trigger_create_sql = """
+    CREATE FUNCTION base_translation_entity_document_update() RETURNS TRIGGER AS $$
```

Comment:
I'd change the name of this function, let's call it `base-entity_document_update`.